tests/sealKey/enc/enc.cpp

```diff
@@ -286,17 +219,14 @@ bool TestOEGetSealKey()
 
         // Modify the isv_svn of key request to invalid and verify the function
         // can't get seal key.
-        sgx_key_request_t* key_request = (sgx_key_request_t*)key_info;
+        sgx_key_request_t* key_request = (sgx_key_request_t*)key_info_ptr;
         uint16_t cur_isv_svn = key_request->isv_svn;
         key_request->isv_svn = 0XFFFF;
-        ret = oe_get_seal_key_v1(
-            key_info, key_info_size, third_key_buffer, &third_key_buffer_size);
-        if (ret != OE_INVALID_ISVSVN)
-        {
-            return false;
-        }
         ret = oe_get_seal_key_v2(
-            key_info, key_info_size, &third_key_ptr, &third_key_ptr_size);
+            key_info_ptr,
+            key_info_ptr_size,
+            &third_key_ptr,
+            &third_key_ptr_size);
         if (ret != OE_INVALID_ISVSVN)
         {
             return false;
```

Comment:
if ret is OE_OK, then you need to free third_key_ptr before returning

Reply:
Fixed